app/src/main/java/it/kapfer/digitalesamt/liberator/ModuleMain.kt

```diff
@@ -110,7 +124,6 @@ class ModuleMain : IXposedHookZygoteInit, IXposedHookLoadPackage {
         // Hook HomeFragment's getHasAttestationCapabilities() method
         XposedHelpers.findAndHookMethod(HOMEFRAGMENT_CLASS, lpparam.classLoader, "getHasAttestationCapabilities", XC_MethodReplacement.returnConstant(true))
 
-        /*
         // TODO: find out how attestation in edu.digicard works
```

Comment:
Since you've uncommented this part again, can we assume that you found out how attestation in *edu.digicard* works and the app runs now? Is this `TODO` therefore done?

Reply:
I don't really know why I commented this part in the first place, as this is required to fix the app (and it works). Please see my comment in #9